ros__parameters:
  package_name: 'reinforcement_learning'
  use_sim: True
  logging: False

  sim_param:
    pose_topic: '/ego_racecar/odom'
    odom_topic: '/ego_racecar/odom'
    scan_topic: '/scan'
    
  real_param:
    pose_topic: '/pf/pose/odom'
    odom_topic: '/odom'
    scan_topic: '/scan'
    
  logger_param:
    count_lap: True
